client/state/first-view/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function isViewVisible( state, view ) {
+	return -1 !== state.firstView.visible.indexOf( view );
+}
+
+export function isViewDismissed( state, view ) {
+	return -1 !== state.firstView.dismissed.indexOf( view );
+}
```

Comment:
For both of these, also need to take into account whether the section actually has a first view. Something like:

```
export function isSectionFirstViewable( view ) {
	return some( FIRST_VIEW_SECTIONS, section => section === view );
}

export function isViewVisible( state, view ) {
	return isSectionFirstViewable( view ) && -1 !== state.firstView.visible.indexOf( view );
}

export function isViewEnabled( state, view ) {
	return isSectionFirstViewable( view ) && -1 === state.firstView.disabled.indexOf( view );
}
```